app/src/main/java/com/android/sample/ui/account/EditAccountScreen.kt

```diff
@@ -0,0 +1,451 @@
+ * @param modifier the modifier which will be passed to the box containing the image of the profile
+ *   picture and the button to change the picture
+ * @param photoPickerLauncher the launch activity when we choose a photo from the gallery to be used
+ *   as the profile picture
+ */
+@Composable
+private fun ProfilePicture(
+    newProfilePictureImageBitmap: ImageBitmap?,
+    modifier: Modifier = Modifier,
+    photoPickerLauncher: ManagedActivityResultLauncher<PickVisualMediaRequest, Uri?>
+) {
+  Box(modifier = modifier.aspectRatio(1f), contentAlignment = Alignment.Center) {
+    if (newProfilePictureImageBitmap == null) {
+      Image(
+          painter = painterResource(id = R.drawable.account),
+          contentDescription = PROFILE_PICTURE_CONTENT_DESCRIPTION,
+          contentScale = ContentScale.FillBounds,
+          modifier =
+              Modifier.fillMaxSize()
+                  .clip(CircleShape)
+                  .border(2.dp, Color.Black, shape = CircleShape)
+                  .testTag(PROFILE_PICTURE_TAG))
+    } else {
+      Image(
+          bitmap = newProfilePictureImageBitmap,
+          contentDescription = PROFILE_PICTURE_CONTENT_DESCRIPTION,
+          contentScale = ContentScale.FillBounds,
+          modifier =
+              Modifier.fillMaxSize()
+                  .clip(CircleShape)
+                  .border(2.dp, Color.Black, shape = CircleShape)
+                  .testTag(PROFILE_PICTURE_TAG))
+    }
+    Box(
+        modifier =
+            Modifier.align(Alignment.BottomEnd)
+                .size(50.dp)
+                .offset(x = (-7.5).dp, y = (-7.5).dp)
+                .background(MaterialTheme.colorScheme.background, shape = CircleShape)
+                .border(1.dp, Color.Black, CircleShape)
+                .clickable { openGallery(photoPickerLauncher) }) {
+          Icon(
+              imageVector = Icons.Default.CameraAlt,
+              contentDescription = CHANGE_PROFILE_PICTURE_BUTTON_DESCRIPTION,
+              modifier =
+                  Modifier.fillMaxSize().padding(5.dp).testTag(CHANGE_PROFILE_PICTURE_BUTTON_TAG),
+              tint = Color.Black)
+        }
+  }
+}
+
+/**
+ * Function that creates an input text field composable for a given field
+ *
+ * @param boxName the name given to the text field, which will be displayed above the value inside
+ *   it
+ * @param boxValue the value which will be contained in the text field
+ * @param readOnly [Boolean] indicating if the field is read only or not
+ * @param onValueChange action to be performed when the value of the field changes
+ */
+@Composable
+private fun InputTextBox(
+    boxName: String,
+    boxValue: String,
+    readOnly: Boolean,
+    onValueChange: (String) -> Unit
+) {
+  OutlinedTextField(
+      value = boxValue,
+      onValueChange = onValueChange,
+      modifier = Modifier.width(350.dp).testTag("$boxName text field"),
+      textStyle = TextStyle(fontSize = 16.sp),
```

Comment:
You need to use styles from `MaterialTheme.typography`, preferably one of the smaller ones, because the 16.sp one is too big for smaller phones.

Reply:
Nice catch. To be honest, I forgot to test this. I changed it to `MaterialTheme.typography.bodySmall`